addons/web_editor/static/src/js/wysiwyg/widgets/media.js

```diff
@@ -87,6 +87,18 @@ var SearchableMediaWidget = MediaWidget.extend({
      */
     search: function (needle) {},
 
+    //--------------------------------------------------------------------------
+    // Private
+    //--------------------------------------------------------------------------
+
+    /**
+     * Renders thumbnails for the attachments.
+     *
+     * @abstract
+     * @returns {Promise}
+     */
+    _renderImages: function () {},
```

Comment:
An abstract `_renderImages` in something which is not necessarily for images... can't you rename in something more  generic? `_renderMedias` I guess?

Reply:
Renamed to `_renderThumbnails`